tests/test-infra/azure-all.bicep

```diff
@@ -78,9 +82,33 @@ module windowsCluster 'azure.bicep' = {
     namePrefix: '${namePrefix}w'
     location: location2
     enableWindows: true
+    enableArm64 : false
     diagLogAnalyticsWorkspaceResourceId: diagLogAnalyticsWorkspaceResourceId
     diagStorageResourceId: diagStorageResourceId
     enableCosmosDB: enableCosmosDB
     enableServiceBus: enableServiceBus
   }
 }
+
+// Deploy the Arm cluster in the third location
+resource linuxResources 'Microsoft.Resources/resourceGroups@2020-10-01' = {
+  name: 'Dapr-E2E-${namePrefix}l'
```

Comment:
Here too, needs a different suffix, not `l` which is used by Linux. For example `la`?